src/main/java/org/spongepowered/common/registry/SpongeGameRegistry.java

```diff
@@ -397,7 +394,7 @@
     private final Map<String, CookedFish> cookedFishMappings = Maps.newHashMap();
     private final Map<String, DyeColor> dyeColorMappings = Maps.newHashMap();
     private final Map<String, Art> artMappings = Maps.newHashMap();
-    private final Map<String, EntityType> entityTypeMappings = Maps.newHashMap();
+    protected final Map<String, EntityType> entityTypeMappings = Maps.newHashMap();
```

Comment:
Maybe a protected method?
That way you can apply some verification to the params.